src/Microsoft.AspNet.Diagnostics/Views/RuntimeInfoPage.cshtml

```diff
@@ -14,6 +14,9 @@
 
     public RuntimeInfoPageModel Model { get; set; }
 }
+@{
+    Response.ContentType = "text/html; charset=utf-8";
+}
 <!DOCTYPE html>
```

Comment:
Can you run the output of the *rendered* page through the [W3C validator](http://validator.w3.org/)? I suspect this page won't validate because the DOCTYPE isn't at the top of the page. (Razor will render a few empty lines before it.)

If the page doesn't validate, please log a new bug with the validation errors and we can triage it separately.